src/torchmetrics/metric.py

```diff
@@ -569,6 +573,33 @@ def plot(self, *_: Any, **__: Any) -> Any:
         """Override this method plot the metric value."""
         raise NotImplementedError
 
+    def _plot(self, val: Union[Tensor, Sequence[Tensor], None] = None, ax: Optional[_AX_TYPE] = None) -> _PLOT_OUT_TYPE:
+        """Plot a single or multiple values from the metric.
+
+        Args:
+            val: Either a single result from calling `metric.forward` or `metric.compute` or a list of these results.
+                If no value is provided, will automatically call `metric.compute` and plot that result.
+            ax: An matplotlib axis object. If provided will add plot to that axis
+
+        Returns:
+            Figure and Axes object
+
+        Raises:
+            ModuleNotFoundError:
+                If `matplotlib` is not installed
+        """
+        val = val or self.compute()
+        fig, ax = plot_single_or_multi_val(
+            val,
+            ax=ax,
+            higher_is_better=self.higher_is_better,
+            name=self.__class__.__name__,
+            lower_bound=self._plot_lower_bound,
```

Comment:
can we make these not protected but public? They are kind of public API similar to `higher_is_better`

Reply:
personaly I would keep this and protected so user shall not edit it... well we can ban these from editing, then set it public :)

Reply:
check, mind have look :)